### PR TITLE
Dynamic app logos

### PR DESCRIPTION
**Pull request by saurabh6790**

### CODE REVIEW

File: dashboard/src/views/partials/LoginBox.vue [hunk 1]
```diff
@@ -27,22 +27,20 @@
 			</div>
 		</div>
 		<div class="absolute bottom-4 z-[1] flex w-full justify-center">
-			<FrappeLogo class="h-4" />
+			<BrandLogo :footer="1" />
```
Comment (BreadGenie) [inline dashboard/src/views/partials/LoginBox.vue]: not very explicit with using 1
```suggestion
			<BrandLogo :footer="true" />
```

File: dashboard/src2/components/BrandLogo.vue [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import FrappeLogo from '@/components/icons/FrappeLogo.vue';
+import { getBrandInfo } from '../data/branding';
+
+export default {
+	name: 'BrandLogo',
+	components: {
+		FCLogo,
+		FrappeLogo
+	},
+	props: ['header', 'footer'],
```
Comment (BreadGenie) [inline dashboard/src2/components/BrandLogo.vue]: on 2nd thought this should've been a type prop with options being header and footer. Take a look at [AlertBanner.vue](https://github.com/frappe/press/blob/master/dashboard/src2/components/AlertBanner.vue) file

Then we can do
```
<BrandLogo type="header" />
```

**Review by BreadGenie — CHANGES_REQUESTED** [review]
A minor change needed. Otherwise looks good.

**Comment by rmehta** [comment]
PR description, screenshots? @saurabh6790 - This is not 2014!

**Pull request closed without merging**